### ISSUE TITLE
Font Weight problem

### ISSUE THREAD

**Issue by Shaglock** [dqisme/Hactor#1]
Hello! I have installed this font and when I am using it in VSCode it feels like font-weight is bold and I can't change it through `editor.fontWeight`. Maybe I'm doing something wrong? I'm attaching screenshots of my before\after changing to Hactor. 

![image](https://user-images.githubusercontent.com/15055321/86595753-d86ec780-bfa1-11ea-9efd-a296999624d4.png)
![image](https://user-images.githubusercontent.com/15055321/86595779-e1f82f80-bfa1-11ea-9760-f782d7bd4c4b.png)
![image](https://user-images.githubusercontent.com/15055321/86595796-ef151e80-bfa1-11ea-90ae-13bbb000bd0d.png)

**Comment by dqisme** [comment]
Hey, thanks for raising problems.
I tried to reproduce the issue you posted here.
First, would you please try with the `fontWeight` as `bold`? Because in general `100` means lighter than the `normal` (you can learn [here](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)).
Second, please make sure you have installed all the font files in your system. There should be 4 files in total (2 for normal and 2 for bold).
And also the thickness is different between two fonts, even they are all in `normal` weight.

**Comment by Shaglock** [comment]
Hey! Thanks for your response. 

> First, would you please try with the fontWeight as bold? Because in general 100 means lighter than the normal (you can learn here).

I tried every possible value, nothing changes. 

>Second, please make sure you have installed all the font files in your system. There should be 4 files in total (2 for normal and 2 for bold).

Fonts are definitely installed in my system, and they work in VSCode (italic comments works exactly as your screenshot) 
```bash
$ fc-list | grep Hactor
/home/shaglock/.fonts/Hactor-Bold-Italic.ttf: Hactor:style=Bold Italic
/home/shaglock/.fonts/Hactor-Regular.ttf: Hactor:style=Regular
/home/shaglock/.fonts/Hactor-Italic.ttf: Hactor:style=Italic
/home/shaglock/.fonts/Hactor-Bold.ttf: Hactor:style=Bold
```
![image](https://user-images.githubusercontent.com/15055321/86811198-4ee4f580-c086-11ea-970c-94d2e78bef17.png)
I tried installing in several ways, but it does not help with this issue.


> And also the thickness is different between two fonts, even they are all in normal weight.

I understand that, but this font is really bolder than other fonts. I saw similar issues with Cascadia Code font (https://github.com/microsoft/cascadia-code/issues/25), maybe this is kind of the same issue. Even in your screenshot in README font is way lighter than mine.

**Comment by lfom** [comment]
I also think that there is a problem, please see the picture below (picking font in latest Gedit):

![Screenshot from 2020-07-24 18-16-24](https://user-images.githubusercontent.com/15659999/88436119-0180bc00-cdda-11ea-824d-fd16644dbfc0.png)

**Comment by Shaglock** [comment]
> I also think that there is a problem, please see the picture below (picking font in latest Gedit):
> 
> ![Screenshot from 2020-07-24 18-16-24](https://user-images.githubusercontent.com/15659999/88436119-0180bc00-cdda-11ea-824d-fd16644dbfc0.png)

Yeah, same problem Regular looks like Bold. But you have 2 bold versions and 2 bold italic for some reason.

**Comment by lfom** [comment]
Yes, I noticed that too. But I have installed only the 4 (four) TTF files provided by @dqisme. Yesterday I tried to fix that with FonfForge but I wasn't successful. I don't know much about font design/creation, so hopefully someone can fix this issue. I think it is related to the font weight problem, all of them use "medium" as weight value.

**Comment by lfom** [comment]
I think I have found the problem: all the files are set as medium and weight (500). Then I changed to the same settings as the original Hack files using FontForge: _Regular_ and _Italic_ to **Book** and **Regular (400)**, and _Bold_ and _Bold Italic_ to **Bold** and **Bold (700)** and it is working fine here. Screenshots for the Regular font:

![Screenshot from 2020-08-03 04-38-58](https://user-images.githubusercontent.com/15659999/89158850-0c0f2400-d545-11ea-88cb-ee66c623de39.png)
![Screenshot from 2020-08-03 04-42-47](https://user-images.githubusercontent.com/15659999/89158854-0dd8e780-d545-11ea-95e9-a713df734759.png)

**Comment by Shaglock** [comment]
> I think I have found the problem: all the files are set as medium and weight (500). Then I changed to the same settings are the original Hack files using FontForge: _Regular_ and _Italic_ to **Book** and **Regular (400)**, and _Bold_ and _Bold Italic_ to **Bold** and **Bold (700)**. Screenshots for the Regular font:

Thank you for your investigation, I will try to apply the same fix to my PC later this week.

**Comment by dqisme** [comment]
Thank you for the solution. I will update all the files to the repository as well.
________________________________

**Comment by lfom** [comment]
Probably Style Map needs to be configured properly too (it is wrong for the files other than Regular):

![Screenshot from 2020-08-03 14-54-33](https://user-images.githubusercontent.com/15659999/89212452-d34d6a00-d599-11ea-844f-23d98cfcc171.png)

**Comment by lfom** [comment]
Here are the fonts with the width fixed for Italics and BoldItalics as explained in the README (space & nbspc width=662). I have tested them in VS Code and I didn't see the selection problem (I use a them with comments in italics).

I used the latest versions of both Hack and Victor Mono, and MediumItalic for the Italic version and it seems to be closer to the weigh of Hack Regular. The font is correctly detected as monospace and can be used in a terminal emulator, and even seems to have the powerline glyphs (and a few more in the Hack one).

Fell free to delete this post if you test the font and decide to distribute it.

[Hactor-Width-Fixed.zip](https://github.com/dqisme/Hactor/files/5019641/Hactor-Width-Fixed.zip)
